local-dev/templates/firecloud-orchestration.conf

```diff
@@ -132,21 +132,6 @@ shibboleth {
 	publicKeyUrl = "https://broad-shibboleth-prod.appspot.com/dev/public-key.pem"
 }
 
-elasticsearch {
-	urls = "elasticsearch5a1.dsde-dev.broadinstitute.org:9300,elasticsearch5a2.dsde-dev.broadinstitute.org:9300,elasticsearch5a3.dsde-dev.broadinstitute.org:9300"
-	clusterName = "elasticsearch5a"
-	index = "library"
-	ontologyIndex = "ontology"
-	discoverGroupNames = ["all_broad_users","demo_users","GP_DataDelivery_TaskTeam"]
-	shareLogIndex = "sharelog"
-	maxAutocomplete = 1000
-}
-
-duos {
-    baseOntologyUrl = "https://consent-ontology.dsde-dev.broadinstitute.org"
-    dulvn = 4
-}
-
```

Comment:
Once this PR is deployed to prod, I'll follow up with another PR to remove these config stanzas from terra-helmfile.